financial-connections/src/main/java/com/stripe/android/financialconnections/features/bankauthrepair/BankAuthRepairScreen.kt

```diff
@@ -6,11 +6,14 @@ import com.airbnb.mvrx.compose.collectAsState
 import com.airbnb.mvrx.compose.mavericksViewModel
 import com.stripe.android.financialconnections.features.common.SharedPartnerAuth
 import com.stripe.android.financialconnections.features.partnerauth.SharedPartnerAuthState
+import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest.Pane
 
 @Composable
 internal fun BankAuthRepairScreen() {
     // step view model
-    val viewModel: BankAuthRepairViewModel = mavericksViewModel()
+    val viewModel: BankAuthRepairViewModel = mavericksViewModel(
+        argsFactory = { BankAuthRepairViewModel.Args(Pane.BANK_AUTH_REPAIR) }
+    )
```

Comment:
do we need this?

Reply:
See https://github.com/stripe/stripe-android/pull/8135#discussion_r1538052999.